### PR TITLE
[Doc] PR for Add doc for limitation of using same custom Prometheus port under Platform HA

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-5079

Preview: https://deploy-preview-13697--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/manage-deployments/high-availability/#limitations

### CODE REVIEW

**Review by sb-yb — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 1]
```diff
@@ -110,3 +104,11 @@ To remove a standby instance from a high-availability cluster, you need to remov
...
 
 After you have returned a standby instance to standalone mode, the information on the instance is likely to be out of date, which can lead to incorrect behavior. It is recommended to wipe out the state information before using it in standalone mode. For assistance with resetting the state of a standby instance that you removed from a high-availability cluster, contact Yugabyte Support.
+
+## Limitations
+
+If you are using custom ports for Prometheus in your YugabyteDB Anywhere installation and the YugabyteDB Anywhere instance is configured for high availability with other YugabyteDB Anywhere instances, then the following limitation applies: 
+
+- All YugabyteDB Anywhere instances configured under high availability must use the same custom port.
+
+  The default Prometheus port for YugabyteDB Anywhere is `9090`. Custom ports are configured through the settings section of the Replicated installer UI available at `https://<yugabyteanywhere-ip>:8800/`.
```
Comment (subramanian-neelakantan) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:114]: 1. Should we instead point to this page https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/install-software/default/ for "How to access the replicated settings page to customise the port configuration"? This page currently does not show a screenshot of the custom port settings. However, a future edit to this page could consolidate a description of all install time settings that we have allowed on the Replicated UI.
2. Note that the Replicated installer UI is **normally** available at `https://<yugabyteanywhere-ip>:8800`. The port of the replicated UI itself can be configured to any other port, say 7700, during a replicated install using a command like this.
```
curl -sSL https://get.replicated.com/docker | sudo bash -s ui-bind-port=7700
```
In such a case the Replicated UI would have to be accessed at `https://<yugabyteanywhere-ip>:7700`.

**Review by subramanian-neelakantan — APPROVED** [review]
Minor suggestion. Feel free to address (if required) and submit. Thanks for doing this so quickly.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md]: Added link and changed "UI is typically available"

**Review by sb-yb — COMMENTED** [review]
I think one way is that active instance can send alert if secondary port values do not match-up.
I will file  bug so that we actually have better error message in UI for this instead of relying solely on docs.

**Pull request merged**